src/cephadm/cephadm

```diff
@@ -1452,6 +1580,12 @@ def get_container_mounts(fsid, daemon_type, daemon_id,
         data_dir = get_data_dir(fsid, daemon_type, daemon_id)
         mounts.update(NFSGanesha.get_container_mounts(data_dir))
 
+    if daemon_type == CephIscsi.daemon_type:
+        assert daemon_id
+        data_dir = get_data_dir(fsid, daemon_type, daemon_id)
```

Comment:
minor nit - but we've duplicated this line in each of the `if` blocks .. maybe it should move to the top of the func?